### DISCUSSION TITLE
Welcome to data_challenge1 Discussions!

### DISCUSSION THREAD

**Discussion by lwpiotr** [grand-mother/data_challenge1#1] [Announcements]
## 👋 Welcome!
We will be using the Discussions module of Github for sharing general ideas and concerns regarding Data Challenge 1 and Pre-Data Challenge.

If you have a specific issue or a request, please use the "Issues" module. This will make issue tracking and bug fixing much easier.

**Comment by rameshkoirala** [comment]
In an effort to test codes after data_challenge1, I am trying to run grandlib from docker and looks like the latest version of grandlib in docker is 7 months old. Can we update grandlib in docker (@Jean-Marc)? If it is time-consuming and too much work, I will directly pull grandlib from github and start using it. Please let me know what you think.

**Comment by luckyjim** [comment]
Hello,
I don't have a docker image specific to this release, 
If specific GRANDLIB env installation  with ROOT is a problem on your computer , you can use docker image grandlib/dev
Cheers
JMC
